queries/asciidoc/highlights.scm

```diff
@@ -0,0 +1,155 @@
+(title0) @markup.heading.1
+
+(title1) @markup.heading.2
+
+(title2) @markup.heading.3
+
+(title3) @markup.heading.4
+
+(title4) @markup.heading.5
+
+(title5) @markup.heading.6
+
+(email) @markup.link.url @markup.link
+
+(author_line
+  ";" @punctuation.delimiter)
+
+(revision_line
+  "," @punctuation.delimiter
+  ":" @punctuation.delimiter)
+
+[
+  (firstname)
+  (middlename)
+  (lastname)
+] @attribute
+
+(revnumber) @number
+
+(revdate) @string.special
+
+(revremark) @string
+
+(table_block_marker) @punctuation.special
+
+(table_cell_attr) @attribute
+
+(table_cell
+  "|" @punctuation.special)
+
+[
+  (breaks)
+  (hard_wrap)
+  (quoted_block_md_marker)
+  (quoted_paragraph_marker)
+  (open_block_marker)
+  (raw_block_marker)
+  (passthrough_block_marker)
+  (quoted_block_marker)
+  (ntable_block_marker)
+  (anno_marker)
+] @punctuation.special
+
+(ntable_cell
+  "!" @punctuation.special)
+
+(task_list_marker_unchecked) @markup.list.unchecked
+
+(task_list_marker_checked) @markup.list.checked
+
+[
+  (list_marker_star)
+  (list_marker_hyphen)
+  (list_marker_dot)
+  (list_marker_digit)
+  (list_marker_geek)
+  (list_marker_alpha)
+] @markup.list
+
+[
+  (line_comment)
+  (comment_block)
+] @comment @spell
+
+[
+  (document_attr_marker)
+  (element_attr_marker)
+] @punctuation.delimiter
+
+(document_attr
+  (attr_name) @property)
+
+(element_attr
+  (attr_value) @attribute)
+
+(block_title
+  (block_title_marker) @punctuation.special) @attribute
+
+(raw_block_body) @markup.raw.block
```

Comment:
This will interfere with injected syntax and should be `@none` or removed entirely.

Reply:
Those are also injected.

Reply:
I understand what you mean. You don't want any rule be covered.